tests/forte/data/data_store_test.py

```diff
@@ -980,7 +990,7 @@ def test_get(self):
                 "forte.data.ontology.top.Annotation", range_annotation=(1, 20)
             )
         )
-        self.assertEqual(len(instances), 2)
+        self.assertEqual(len(instances), 3)
```

Comment:
could you remind me why this len changes?

Reply:
I believe this was because of an extra `Annotation` entry that was added [here](https://github.com/Pushkar-Bhuse/forte/blob/93d198e4a25fceb8f2ecd7c1ccb5fad09c23db74/tests/forte/data/data_store_test.py#L233) as a part of the test cases

Reply:
what I don't understand is there is no change of the test case in this PR, but there is an assertion change, which makes me feel like the implementation is the reason

Reply:
I think there should be only two instances as only `ref2` and `ref5` are in the range `(1,20)`

Reply:
Yes that is correct. The new implementation handles that.